diesel_derives2/src/field.rs

```diff
@@ -42,6 +47,13 @@ pub enum FieldName {
 
 impl quote::ToTokens for FieldName {
     fn to_tokens(&self, tokens: &mut quote::Tokens) {
+        use proc_macro2::{Spacing, TokenNode, TokenTree};
+
+        // https://github.com/rust-lang/rust/issues/47312
+        tokens.append(TokenTree {
```

Comment:
[`quote_spanned!`](https://docs.rs/quote/0.4/quote/macro.quote_spanned.html) can make this more concise but it is up to you whether you prefer the explicit way or the concise way.

```rust
quote_spanned!(call_site=> .)
```